packages/core/src/lib/collect-and-persist.ts

```diff
@@ -1,21 +1,17 @@
-import {
-  collect,
-  CollectOptions,
-  logPersistedResults,
-  persistReport,
-} from '@quality-metrics/core';
+import { collect, CollectOptions } from './collect';
+import { name, version } from '../../package.json';
```

Comment:
This will mean the package name and version in our reports will be for `@code-pushup/core`, not `@code-pushup/cli`. I can't decide if that's better or worse, honestly, so happy to keep it as is, just thought I'd flag it to make sure it's intentional :man_shrugging:

Reply:
added docs